onnxruntime/python/tools/transformers/float16.py

```diff
@@ -334,11 +344,19 @@ def convert_float_to_float16(
                     for i, input_name in enumerate(n.input):
                         if input_name in fp32_initializers:
                             # For Resize/GroupNorm, only the first input can be float16
-                            use_fp32_weight = is_node_blocked or (
-                                i in ALWAYS_FLOAT_INPUTS.get(n.op_type, [])
-                                and i not in force_fp16_inputs_dict.get(n.op_type, [])
-                            )
-                            fp32_initializers[input_name].add_node(n, use_fp32_weight)
+                            if i in ALWAYS_FLOAT_INPUTS.get(n.op_type, []) and i not in force_fp16_inputs_dict.get(
+                                n.op_type, []
+                            ):
+                                dtype = TensorProto.FLOAT
+                            elif is_node_blocked:
+                                dtype = (
+                                    TensorProto.BFLOAT16
+                                    if (use_bfloat16_as_blocked_nodes_dtype and n.op_type not in BF16_OP_BLACK_LIST)
```

Comment:
I found that `use_bf16=use_bfloat16_as_blocked_nodes_dtype and n.op_type not in BF16_OP_BLACK_LIST`, so it is better we somehow pass that variable and use here.

Reply:
Undo the change of this file in PR so that we can focus on graph fusion for this PR. I will add a new PR for bf16 conversion later